api/server/sdk/server.go

```diff
@@ -58,7 +63,17 @@ type ServerConfig struct {
 type Server struct {
 	*grpcserver.GrpcServer
 
-	restPort             string
+	restPort            string
+	onceSetCluster      sync.Once
```

Comment:
sync.Once not used anywhere

Reply:
oh, nice catch, i was trying different algorithms. I will remove it